libbeat/tests/system/template/template.go

```diff
@@ -30,7 +30,7 @@ import (
 // MaxDefaultFieldLength is the limit on the number of default_field values
 // allowed by the test. This is less that the 1024 limit of Elasticsearch to
 // give a little room for custom fields and the expansion of `fields.*`.
-const MaxDefaultFieldLength = 1000
+const MaxDefaultFieldLength = 1010
```

Comment:
I'm not sure about this change. It was complaining that there were 1006 fields after the ECS update.

Will exceed 1000 have an impact or as long as it's below 1024 we're safe?

Reply:
We configure the `total_fields.limit` per index to 10000. The default_fields limit is impacted by the `indices.query.bool.max_clause_count` setting (default 1024, but not modifyable via templates).

Reply:
@andrewkroh I think touched this in the past?

Reply:
One option to deal with this would be to add `default_field: false` to the new fields being added in ECS. This problem will keep occurring each time new fields are added and this could be the solution until we move to a better indexing strategy. Bumping the limit up won't work for much longer.